frontend/components/bc/premium/BcPremiumModal.vue

```diff
@@ -14,18 +16,24 @@ const hide = () => {
 onMounted(() => {
   setHeader($t('premium.title'), true)
 })
-
 </script>
 
 <template>
   <div class="text">
-    {{ props?.description || $t('premium.description') }}
+    {{ props?.description || $t("premium.description") }}
```

Comment:
Having single quotes in the main script part and double quotes in the inline script seems a bit inconsistent?
![image](https://github.com/user-attachments/assets/2aab0109-a61b-436c-a7dc-7b3b0ded3c0b)

Reply:
could be enforced...is not atm:

https://eslint.vuejs.org/rules/html-quotes.html

have a special look at [avoid escape](https://eslint.vuejs.org/rules/html-quotes.html#double-avoidescape-true) and tell me if I should